README.md

```diff
@@ -1,4 +1,13 @@
+![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
+
+
 [![Travis Badge](https://travis-ci.org/sendgrid/open-source-library-data-collector.svg?branch=master)](https://travis-ci.org/sendgrid/open-source-library-data-collector)
+[![BuildStatus](https://travis-ci.org/sendgrid/sendgrid-nodejs.svg?branch=master)](https://travis-ci.org/sendgrid/sendgrid-nodejs)
```

Comment:
Can you please change `nodejs` to `open-source-library-data-collector` on these lines?

Reply:
Yeah! I just fix those lines on my last commit https://github.com/sendgrid/open-source-library-data-collector/pull/80/commits/e222a6f1ab8cc76bc20f1a0c4a5a048760c8b36e